src/oc/storage/api/entries.clj

```diff
@@ -123,6 +126,12 @@
                       (assoc :auto-share true))]
       (share-entry conn share-ctx (:uuid entry-result)))))
 
+(defn undraft-board [conn user org board]
```

Comment:
Same here.  Undrafting a board should be done when publishing or creating an entry and the code belongs in the entry resource.

Reply:
I don't agree here too, there is no mention of logic behind the board in the entry resource, there is a lot here in the the board api.
Again i think you are suggestion something more then a simple improvement to this code, more a refactor of the storage code.